docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+   'sample-domain2-admin-server'     '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   'sample-domain2-managed-server1'  '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   'sample-domain2-managed-server2'  '1'      'model-in-image:WLS-v1'  'true'  'Running'  
+   
+   
+   @@ [2020-05-13T17:08:34][seconds=155] Info: Success!
+
+   ```
+   {{% /expand%}}
+
+1. After the `sample-domain2` domain is running, you can call its sample web application to verify that it's fully active.
+
+   Send a web application request to the ingress controller for `sample-domain2`:
+
+   ```  
+   $ curl -s -S -m 10 -H 'host: sample-domain2-cluster-cluster-1.mii-sample.org' \
+      http://localhost:30305/myapp_war/index.jsp
+   ```  
+
+   Or, if Traefik is unavailable and your `domain2` Administration Server pod is running, you can run `kubectl exec`:
+
+   ```  
+   $ kubectl exec -n sample-domain1-ns sample-domain2-admin-server -- bash -c \ 
+     "curl -s -S -m 10 http://sample-domain2-cluster-cluster-1:8001/myapp_war/index.jsp"
+   ```  
+
+   You should see something like the following:
+
+   {{%expand "Click here to see the expected web application output." %}}
+
+   ```  
+   <html><body><pre>
+   *****************************************************************
+
+   Hello World! This is version 'v1' of the mii-sample JSP web-app.
+
+   Welcome to WebLogic server 'managed-server1'!
+
+    domain UID  = 'sample-domain2'
+    domain name = 'domain2'
+
+   Found 1 local cluster runtime:
+     Cluster 'cluster-1'
+
+   Found 1 local data source:
+     Datasource 'mynewdatasource': State='Running'
+
+   *****************************************************************
+   </pre></body></html>
+
+   ```  
+   {{% /expand%}}
+
+If you see an error, then consult [Debugging]({{< relref "/userguide/managing-domains/model-in-image/debugging.md" >}}) in the Model in Image user guide.
+
+We will not be using the `sample-domain2` domain again in this sample; if you wish, you can shut it down now by calling `kubectl -n sample-domain1-ns delete domain sample-domain2`.
+
+### Update3 use case
+
+The Update3 use case demonstrates deploying an an updated WebLogic application to the running [Update1 use case](#update1-use-case) domain using an updated Docker image.
+
+In the use case, we will:
+
+ - Create an image `model-in-image:WLS-v2` that is similar to the currently active `model-in-image:WLS-v1` image, but with with the following updates:
+   - An updated web application `v2` at the `myapp-v2` directory path within the WDT application archive instead of `myapp-v1`.
+   - An updated model YAML within the image that points to the new web application path.
+ - Apply an updated domain resource that references the new image while still referencing the original [Update1 use case](#update1-use-case) case secrets and model ConfigMap. 
+
+After the updated domain resource is applied, we expect the operator to:
```

Comment:
"we expect the operator to" sounds like we don't know if it works.
"the operator will" sounds better

Reply:
done